cli/Cargo.toml

```diff
@@ -10,7 +10,7 @@ with_mimalloc = ["mimalloc"]
 [dependencies]
 cairo-lang-starknet = { git = "https://github.com/starkware-libs/cairo", rev = "4afacfd574de1121d61dea28f8c73e1516c7e07d"}
 cairo-lang-casm = { git = "https://github.com/starkware-libs/cairo", rev = "4afacfd574de1121d61dea28f8c73e1516c7e07d"}
-cairo-vm = { git = "https://github.com/lambdaclass/cairo-rs", features = ["cairo-1-hints"], rev = "5db2e656d46a97d89563a74f8fc3abd595e8befb"  }
+cairo-vm = { git = "https://github.com/lambdaclass/cairo-rs", features = ["cairo-1-hints"], rev = "de6a2327b7e629336bc2431787c8e28c3b0f1349" }
```

Comment:
I suggest we use workspace level deps instead. That will make updating __much__ easier.